project.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE = html>
+<html lang="en">
+<head>
+  	<meta charset="utf-8">
+  	<title>Portfolio Project</title>
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link href="normalize.css" type="text/css" rel="stylesheet">
+	<link href="Portfoliostyle.css" type="text/css" rel="stylesheet">	
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+
+<body>
+	<nav>
+		<ul>
+			<li><a href="portfolio.html">Home</a></li>
+			<li><a href="portfolio.html#portfolio">Portfolio</a></li>
+			<li><a href="portfolio.html#aboutme">About Me</a></li>
+			<li><a href="portfolio.html#contact">Contact</a></li>
+		 </ul>
+		 <div><img src="" alt=""/>
+	</nav>
+
+	<div class="container">
+		<img src="projectimg.jpg" alt="Photos of Project">
+		<h2>Project The First</h2>
+		<p>This went well and we were all thrilled.</p>
+	</div>
+	
+</body>
+
+
+
+<footer>
+	<div class="social">
+		<a href="https://twitter.com/jennyat9" class="fa fa-twitter"></a>
+		<a href="https://linkedin.com/in/jennyalltontwist" class="fa fa-linkedin"></a>
+		<a href="https://github.com/jennymegan" class="fa fa-github"></a>
```

Comment:
These media links are visually appearing in the middle of the page - maybe because your footer tag is outside of your body tag?

Reply:
Thank you! Sorted now